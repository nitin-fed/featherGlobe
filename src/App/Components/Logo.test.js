import React from 'react'
import renderer from 'react-test-renderer'
import Logo from './Logo'

describe('Logo', () => {
    it('Should render logo', () => {
        const component = renderer.create(<Logo />)
        const tree = component.toJSON();
        console.log(tree)
        expect(tree).toMatchSnapshot();
    })
})