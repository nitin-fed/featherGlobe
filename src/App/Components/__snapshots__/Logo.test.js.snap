// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should render logo 1`] = `
<div
  className="logo"
>
  <p />
  <img
    alt="FeatherGlobe"
    className="logo"
    src="../images/globe.gif"
  />
</div>
`;
